sympy/utilities/autowrap.py

```diff
@@ -8,14 +8,34 @@
 This module provides a common interface for different external backends, such
 as f2py, fwrap, Cython, SWIG(?) etc. (Currently only f2py and Cython are
 implemented) The goal is to provide access to compiled binaries of acceptable
-performance with a one-button user interface, i.e.
+performance with a one-button user interface, e.g.,
 
     >>> from sympy.abc import x,y
-    >>> expr = ((x - y)**(25)).expand()
-    >>> binary_callable = autowrap(expr)
-    >>> binary_callable(1, 2)
+    >>> expr = (x - y)**25
+    >>> flat = expr.expand()
+    >>> binary_callable = autowrap(flat)
+    >>> binary_callable(2, 3)
     -1.0
 
+Although a SymPy user might primarily be interested in working with
+mathematical expressions and not about the details of wrapping tools
+in order to evaluate expressions numerically which are expensive to
+calculate, the user cannot do so without some understanding of the
```

Comment:
```suggestion
mathematical expressions and not in the details of wrapping tools
needed to evaluate such expressions efficiently in numerical form,
the user cannot do so without some understanding of the
```

Reply:
I'll wait to commit this change to see if there are other suggestions so they can be committed together.